assets/javascript/exports.js

```diff
@@ -75,6 +75,10 @@ const initializeExportRunButton = function (apiUrl, progressUrl) {
         progressBar.setAttribute('value', '100');
         progressMessage.innerText = errorMessage;
         return;
+      }else{  // Initialize progressbar
```

Comment:
nit: could you add some whitespace here? we don't have a linter on this repo yet.

Reply:
Also, I wonder if this is an issue with the CeleryProgressBar upstream package (https://github.com/czue/celery-progress/blob/master/celery_progress/static/celery_progress/celery_progress.js)? It seems funny to me to need to "initialize" this outside of the `initProgressBar` function. It could be worth submitting a PR to that project. I hear the maintainer is quite friendly :wink:

Reply:
yeah sure I can do that.

Reply:
btw, lynt is fixed.